packages/coachmark/src/Coachmark.ts

```diff
@@ -179,9 +178,7 @@ export class Coachmark extends LikeAnchor(Popover) {
     }
 
     private stopPropagationOnHref(event: Event): void {
```

Comment:
If it's not `LikeAnchor` does this method and the associated binding?

Reply:
Right! This is redundant now!